src/test_l1_tvSettings.c

```diff
@@ -15988,8 +15986,8 @@ void test_l1_tvSettings_negative_GetTVGammaTarget (void)
     if (extendedEnumsSupported == true)
     {
         /* Step 08: Calling tvsettings GetTVGammaTarget after TvTerm*/
-        GetTVGammaTarget(colorTempValue, &x_Value, &y_Value);
-        UT_ASSERT_FALSE((x_Value >= 0 && x_Value <= 1.0));
+        result = GetTVGammaTarget(colorTempValue, &x_Value, &y_Value);
+        UT_ASSERT_EQUAL_FATAL(result, tvERROR_INVALID_STATE);
```

Comment:
Dont use UT_ASSERT_EQUAL_FATAL instead use UT_ASSERT_EQUAL

Reply:
addressed in commit id:- 45ba5edef84a528bb3ef3e891c40bc01d1646c0a